protocol-units/bridge/README.md

```diff
@@ -0,0 +1,120 @@
+# Current Contract Deployments
+## Holesky Ethereum Deployments
+
+| Contract                         | Address                                      |
+|-----------------------------------|----------------------------------------------|
+| `AtomicBridgeInitiatorMove.sol`   | 0xb33d9fA868054FD3F8F02ba9Fc04B2a1862Ab855   |
+| `AtomicBridgeCounterpartyMove.sol`    | 0xA6bB0d5eA03f3bEe3510EA3Ec39e85Cd7395Bc37   |
+
+
+## testnet.movementlabs Deployments
+the movement atomic-bridge and all of its modules are deployed under the same package address:
+`0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c`
+
+Verify this by running:
+```
+ movement account list --account 0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c 
+```
+## Running the Atomic Bridge Relayer In Production
+The atomic bridge relayer services requires a `config.json` at the root of the application inside a `./movement`
+
+Set these values to your needs to configure the relayer.
+
+```json
+{
+  "eth": {
+    "eth_rpc_connection_protocol": "http",
+    "eth_rpc_connection_hostname": "0.0.0.0",
+    "eth_rpc_connection_port": 8090,
+    "eth_ws_connection_protocol": "ws",
+    "eth_ws_connection_hostname": "0.0.0.0",
+    "eth_ws_connection_port": 8090,
+    "eth_chain_id": 3073,
+    "eth_initiator_contract": "0xb33d9fA868054FD3F8F02ba9Fc04B2a1862Ab855",
+    "eth_counterparty_contract": "0xA6bB0d5eA03f3bEe3510EA3Ec39e85Cd7395Bc37",
+    "eth_weth_contract": "0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e",
+    "signer_private_key": "<ETH_PRIVATE_KEY>",
+    "time_lock_secs": 60,
+    "gas_limit": 10000000000000000,
+    "transaction_send_retries": 10
+  },
+  "movement": {
+    "movement_signer_address": "<MOVEMENT_PRIVATE_KEY>",
+    "movement_native_address": "0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c",
+    "mvt_rpc_connection_protocol": "http",
+    "mvt_rpc_connection_hostname": "0.0.0.0",
+    "mvt_rpc_connection_port": 30731,
+    "mvt_faucet_connection_protocol": "http",
+    "mvt_faucet_connection_hostname": "0.0.0.0",
+    "mvt_faucet_connection_port": 30732,
+    "mvt_init_network": "custom"
+  },
+  "testing": {
+    "eth_well_known_account_private_keys": [
+     <0th EL FROM ANVIL>,
+     <1st EL FROM ANVIL>,
+      ...
+    ]
+  }
+}
+
+```
+Once the relayer is up and listening for events, you can initiate a transfer.
+
+## Bridging MOVE from Ethereum to Movement without a UI
+
+First we need to create the `preimage` or secret, for the transfer. 
```

Comment:
You can add: This preimage is an arbitrary array of 32 bytes. For each transfer, the premiage should be regenerated and be unique.
to describe what is a preimage.